nb-no.json

```diff
@@ -0,0 +1,120 @@
+{
+   "nb-no":{
```

Comment:
Are you sure the code is `nb-no`?  I would think it would be just `no` for Norway and only need a sub code if there are multiple Norwegian dialects?

Reply:
There are multiple official Norwegian dialects.
nb / nb-no: Norwegian Bokmål
nn / nn-no: Norwegian Nynorsk

Reply:
My translations are Norwegian Bokmål, so should be marked as either "nb" or "nb-no", depending on your conventions.

Reply:
OK I trust you have it right!